src/containers/facility/FacilityOffers.tsx

```diff
@@ -1,15 +1,60 @@
 import { useParams } from 'react-router-dom';
 import { RoomCard } from 'src/components/RoomCard';
 import { useAccommodationsAndOffers } from 'src/hooks/useAccommodationsAndOffers.tsx';
+import { styled, Typography, useTheme } from '@mui/material';
+import { Box } from '@mui/material';
+import { forwardRef } from 'react';
 
-export const FacilityOffers = () => {
+interface SearchCriteriaAndResult {
+  rooms?: number;
+  guests: number;
+  startDate: string;
+  nights: number;
+  roomsAvailable?: number;
+}
+
+const FacilityOffersContainer = styled(Box)(({theme}) => ({
+  marginBottom: theme.spacing(2.5),
+  padding: theme.spacing(2.5)
+}));
+
+const FacilityOffersTitle = ({
+  rooms,
+  roomsAvailable,
+  guests,
+  startDate,
+  nights
+}: SearchCriteriaAndResult) => {
+
+  const SubHeader = styled(Box)(() => ({}));
+  const theme = useTheme();
+
+  return (
+    <Box marginBottom={theme.spacing(2.5)}>
+      <Typography marginBottom={theme.spacing(6)}>Available Rooms</Typography>
+      <SubHeader>
+        Results for {rooms || ''} room, {guests} guests, staying from {startDate} for{' '}
+        {nights} nights: {roomsAvailable} rooms available.
+      </SubHeader>
+    </Box>
+  );
+};
+
+export const FacilityOffers = forwardRef<HTMLDivElement>((_, ref) => {
   const { getAccommodationById, accommodations } = useAccommodationsAndOffers();
   const params = useParams();
   const id: string = params.id as string;
   const accommodation = getAccommodationById(accommodations, id);
 
   return (
-    <>
+    <FacilityOffersContainer ref={ref}>
+      <FacilityOffersTitle
+        rooms={1}
```

Comment:
this is harcoded, is this correct?

Reply:
We do not have these values available here except in the `searchForm`. 
Had a discussion with Batyr about storing most recent search params to be shared in other parts of the application, for example here. He mentioned a discussion about rethinking state management design and I had to keep it in like this.

Reply:
It is possible to get the latest search results with a minor tweak. 

I think it easies to hop on a call, to impelment this together. What you think?